scripts/zkai_native_seq32_attention_mlp_adjacent_label_policy_gate.py

```
@@ -0,0 +1,743 @@
+        raise AdjacentLabelPolicyGateError("source artifact inventory drift")
+    for item in artifacts:
+        artifact = _dict(item, "source artifact")
+        path = ROOT / str(artifact.get("path"))
+        raw = read_repo_file(path, "source artifact")
+        if artifact.get("sha256") != sha256(raw) or artifact.get("size_bytes") != len(raw):
+            raise AdjacentLabelPolicyGateError("source artifact digest drift")
+
+
+def validate_mutation_result(result: dict[str, Any]) -> None:
+    if result.get("mutation_names") != list(MUTATION_NAMES):
+        raise AdjacentLabelPolicyGateError("mutation result drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)